### DISCUSSION TITLE
Unable to expand environment variable in host setting: 'https://rinkeby.infura.io/v3/$WEB3_INFURA_PROJECT_ID'

### DISCUSSION THREAD

**Discussion by GhoulZoid** [smartcontractkit/full-blockchain-solidity-course-py#695] [Q&A]
![image](https://user-images.githubusercontent.com/95380582/148192823-8d325040-0c2a-4e93-aa31-3f984bccb9ef.png)
Anyone know the cause of this?

**Comment by cromewar** [comment]
Hello @GhoulZoid it means is not finding the project id on your env variables, so please remember to export it from the `.env` again, as you are on windows try adding it using the advanced configuration on windows.
